takwimu/sql/population_residence.sql

```diff
@@ -224,30 +224,6 @@ level1	KE_1_027	2009	rural	548620
 level1	KE_1_038	2009	rural	380517
 level1	KE_1_008	2009	rural	565086
 level1	KE_1_024	2009	rural	469994
-country	ET	2009	urban	11956170
```

Comment:
I've noticed you're removing `ET` data. Any reason?

Reply:
Resolved